assignments/assignment03.rb

```diff
@@ -7,6 +7,22 @@
 
 # re-implement titleize and palindrome? as methods on String
 
+class String
+  def titleize(s)
```

Comment:
titleize won't accept a parameter, since it's operating on the instance of the string itself